confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java

```diff
@@ -101,7 +101,7 @@ public class ConfigNodeConfig {
   private int defaultDataRegionGroupNumPerDatabase = 2;
 
   /** The maximum number of DataRegions expected to be managed by each DataNode. */
-  private double dataRegionPerProcessor = 1.0;
+  private double dataRegionPerDataNode = 5.0;
```

Comment:
Change to 3 will be more safe

Reply:
<img width="1452" alt="image" src="https://user-images.githubusercontent.com/32640567/226291277-8ec156a0-29ea-4617-9d0a-79eb6239c154.png">
We tested the write performance comparison of different number of regions (1/2/4/8/16/32) under a typical load on a fit machine (16core), and felt that it was not always optimal when the value was small

Reply:
That's right. Because our BM uses InsertTablet to do the test, which relies less on disk but more on CPU